src/main/scala/com/evolution/scache/Cache.scala

```diff
@@ -348,6 +351,13 @@ trait Cache[F[_], K, V] {
 
 object Cache {
 
+  sealed trait Modification[+F[_], +V]
```

Comment:
I'd rename this to `Directive` or something more neutral, as it's not about modifying in all cases…